### ISSUE TITLE
vidoe record support

### ISSUE THREAD

**Issue by 532910** [vector-im/riotX-android#1623]
Selecting the `Camera` from clip menu starts camera in phot only mode:
![image](https://user-images.githubusercontent.com/14617699/86517876-20241080-be35-11ea-8711-993b2e098c8d.png)
![image](https://user-images.githubusercontent.com/14617699/86517882-2adea580-be35-11ea-8c43-66139127cb06.png)

While the same camera app (org.lineageos.snap) launched from the home screen has video button:
![image](https://user-images.githubusercontent.com/14617699/86517907-6f6a4100-be35-11ea-9b16-1becb7e949a1.png)

**Comment by kojid0** [comment]
This might be fixed with #1389

**Comment by 532910** [comment]
#1389 isn't so simple to implement, comparing with just enable video support for external camera.

**Comment by kojid0** [comment]
1389 was labeled with riotx-go-live tag so I think the dev sees a priority in this one, but not urgent anymore to have it done in the next weeks. Let's see how it goes.
Doing work on something that anyways will probably be changed very soon is unnecessary but video record support is needed indeed

**Comment by 532910** [comment]
No one said that the current support for external camera will be removed.

**Comment by kojid0** [comment]
Yes and that's good! In-app cameras usually suffer from worse quality compared to external camera.

**Comment by 532910** [comment]
So there is nothing wrong to add support for video.

**Comment by kojid0** [comment]
No, not at all, just wanted to say that the dev is aware of camera feature lacks and already has plans to do work on it (1389). Therefore I've linked this issue, so he knows that both issues belong to the same thing (camera work).

**Comment by sl5net** [comment]
BTW As soon as that's possible, it will be time for some of my friends to say goodbye to the telegram app. Use less and less.
